src/components/layout/Layout.ts

```diff
@@ -136,6 +136,34 @@ export class Layout {
     }
     this.elements = [];
   }
+
+  duplicateElement(
+    selectedCell: [number, number],
+    hoveredCell: [number, number] | undefined,
+  ) {
+    const selectedCellX = selectedCell[0];
```

Comment:
i'm pretty sure it's [row, column], so technically Y, X rather than X. Y

Reply:
@KittenHero Thanks! I've renamed it appropriately.